bundle/bin/rke2-uninstall.ps1

```diff
@@ -345,6 +365,20 @@ function Remove-Namespace() {
     Invoke-Ctr -cmd "namespace remove $namespace"
 }
 
+function Create-Lockfile() {
+    # We fetch ctr.exe path and place the lock there
+    $command = Get-Command ctr -ErrorAction SilentlyContinue
+    if ($command) {
+        $executablePath = $command.Source
+        $dataBinDirDirectory = Split-Path -Parent $executablePath
+        $lockFilePath = Join-Path -Path $directory -ChildPath "rke2-uninstall.lock"
```

Comment:
Looking back over this with the new changes, I don't see `$directory` being set anywhere, looks like it should be `$dataBinDirDirectory`

Reply:
good catch! Let me retest this carefully again